src/hooks/useFormState/useFormState.ts

```diff
@@ -143,6 +145,7 @@ export const useFormState = <TFieldValues>(
     fields: validatedFields,
     values,
     isDirty,
+    isTouched,
     isValid: hasRunInitialValidation && !validationError,
```

Comment:
@gildub I wonder if instead of having all the buttons use `!form.isTouched && !form.isValid` we should just only consider the form valid if it's been touched? I'm honestly not sure.